socialhome/users/tests/test_views.py

```diff
@@ -390,18 +415,27 @@ def test_get_json_context(self):
         self.assertEqual(
             view.get_json_context(),
             {
-                "currentBrowsingProfileId": None,
+                "currentBrowsingProfileId": request.user.id,
```

Comment:
Same here

Reply:
You can also probably remove the view.followers_count = 3 from this test slightly above line 389.

Reply:
Same here, should not be `user.id`. Also, what is weird that we're using `AnonymousUser`, so this test should probably fail :/ I'll have a quick look at why it's not.

Reply:
Ah, this is why:

```
In [4]: AnonymousUser().id is None
Out[4]: True
```

Still, this would be better more explicit to say `"currentBrowsingProfileId": None,` as it was before.